packages/shared/catalogs/types.ts

```diff
@@ -94,6 +96,9 @@ export type PartialEmote = PartialBy<Omit<Emote, 'baseUrlBundles'>, 'baseUrl'>
 export type PartialItem = PartialWearableV2 | PartialEmote
 type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>
 
+export const isPartialWearable = (partialItem: PartialItem): partialItem is PartialWearableV2 =>
+  !!(partialItem as PartialWearableV2).data
```

Comment:
Good question! I _believe_ it's enough since `isPartialWearable` is used to access the `data` object and the `PartialEmote` won't ever have it so adding the check sounds redundant.